.travis/test.ls

```diff
@@ -0,0 +1,30 @@
+require! \stream-snitch
+pty = require \pty.js
+
+urbit =
+  # TODO abort on failure
+  pty.spawn './urbit' <[-B urbit.pill -A .. -cFI zod zod]> 
+     .on \data -> process.stdout.write it
+     
+urbit.on \exit (code)->
+  console.log "\nnode: urbit exited with code #code\n"
+  process.exit code
+
+fin = no
+urbit.pipe (new stream-snitch /dojo> /g).on \match ->
+    return if fin
+    fin := yes
+    console.log "\n\n---\nnode: got dojo!\n---\n\n"
+    set-timeout (-> process.exit 0), 1000 # should probably test further
```

Comment:
There should be a comment explaining why we do this in a setTimeout instead of just immediately (it's to give Urbit time to finish printing whatever it's printing).

Reply:
Also, this is a time-based heuristic approach. Maybe we could make this more rigorous.